### PR TITLE
Changed soft assert to thread-independent in L10N

### PR DESCRIPTION
**Pull request by akamarouski**

### CODE REVIEW

File: docs/advanced/localization.md [hunk 1]
```diff
@@ -106,3 +106,23 @@ xpath = "//*[text()='{L10N:HomePage.welcomeText}'"
 # actual value at run-time:
 xpath = "//*[text()='Willkommen bei Wikipedia'"
 ```
+## Creation of multi-language tests
+If one or a group of tests checks several language versions of the site, then you need to do the following:
+1. (Optional) if you need to change the language version of the site, you can overwrite the locale parameter:
```
Comment (vdelendik) [inline docs/advanced/localization.md:111]: optional usually is written in square brackets: `[Optional]`

File: carina-utils/src/main/java/com/qaprosoft/carina/core/foundation/utils/resources/L10N.java [hunk 2]
```diff
@@ -55,12 +55,16 @@ public class L10N {
...
+    private static SoftAssert mistakes;
+
 
     /**
-     * Load L10N resource bundle.
+     * Load L10N resource bundle corresponding to a specific locale.
+     * If called setLocale function in the test, must be called to reload resources
      */    
     public static void load() {
+        // #1679 fix
```
Comment (vdelendik) [inline carina-utils/src/main/java/com/qaprosoft/carina/core/foundation/utils/resources/L10N.java:66]: replace everywhere to `// #1679: L10N: made assertion threads dependent`

File: docs/advanced/localization.md (hunk 1 above)
Comment (akamarouski) [reply docs/advanced/localization.md]: done

File: carina-utils/src/main/java/com/qaprosoft/carina/core/foundation/utils/resources/L10N.java (hunk 2 above)
Comment (akamarouski) [reply carina-utils/src/main/java/com/qaprosoft/carina/core/foundation/utils/resources/L10N.java]: done

**Pull request merged**